katello/katello-service

```diff
@@ -73,6 +77,18 @@ def list_services
   puts `chkconfig --list 2>&1 | grep '#{regex}'`
 end
 
+def status_services(action)
+  if `which systemctl 2>&1` && $?.success? && !(@options.include?(:verbose))
+    regex = services_by_priority.map { |service| "^[^a-z_-]*#{service}.service" }.join('\|')
+    puts `systemctl list-units --all \*.service | grep -e '#{regex}' -e '^ *UNIT'`
+  else
+    services_by_priority.each do |service|
+      puts "#{COMMAND} #{service} #{action}"
+      puts `#{COMMAND} #{service} #{action}`
```

Comment:
I agree, that would be something good to investigate.  I think this would be something that would be more beneficial when we can start utilizing systemctl for all commands.